test/ctia/http/routes/observable/judgements_test.clj

```diff
@@ -65,5 +87,14 @@
           (get "ctia/ip/10.0.0.1/judgements"
                :headers {"api_key" "45c1f5e3f05d0"})]
       (is (= 200 status))
-      (is (= #{"judgement-1" "judgement-2"}
-             (set (mapcat :external_ids judgements)))))))
+      (is (= #{"judgement-1" "judgement-2" "judgement-3"}
+             (set (mapcat :external_ids judgements))))))
+
+  (testing "GET /:observable_type/:observable_value/judgements?sort_by=disposition%3Aasc%2Cvalid_time.start_time%3Adesc"
```

Comment:
Maybe think about putting the query parameters into the get parameters so its easier to read, so (get ... :query-params {...}).